Libraries/Components/ProgressBarAndroid/ProgressBarAndroid.android.js

```diff
@@ -84,9 +82,9 @@ export type ProgressBarAndroidProps = $ReadOnly<{|
  */
 const ProgressBarAndroid = (
   props: ProgressBarAndroidProps,
-  forwardedRef: ?React.Ref<'AndroidProgressBar'>,
+  forwardedRef: ?React.Ref<ProgressBarAndroidType>,
```

Comment:
I don't think you need this type exported from this file. You should just be able to do `React.Ref<ProgressBarAndroidNativeComponent>`, or if that doesn't work then `React.Ref<typeof ProgressBarAndroidNativeComponent>`

Reply:
Resolved